src/WorkerChannel.ts

```diff
@@ -101,6 +101,13 @@ export class WorkerChannel implements IWorkerChannel {
     if (this._v1WorkerBehavior) {
       if (version.startsWith("v12.")) {
         systemWarn("The Node.js version you are using (" + version + ") is not fully supported with Azure Functions V2. We recommend using one the following major versions: 8, 10.");
+      } else if (version.startsWith("14.")) {

```

Comment:
```suggestion
      } else if (version.startsWith("v14.")) {

```

![image](https://user-images.githubusercontent.com/3982077/93387949-d84f3b80-f81e-11ea-8294-b8f8343b659e.png)